files/jsbarcode/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "JsBarcode",
+  "repo": "lindell/JsBarcode",
+  "files": {
+    "include": ["bin/browser/*.min.*", "bin/browser/barcodes/*.min.*"]
```

Comment:
The bot is now saying that I should use `././barcodes/*.min.*`. That seems very unnecessary but it's maybe what I should do?